src/Microsoft.DocAsCode.MarkdownLite/JsonTokenTreeRenderer.cs

```diff
@@ -121,12 +126,12 @@ public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownTableBloc
                     {
                         childContent += renderer.Render(item);
                     }
-                    rowContent += this.Insert("RowItem", childContent, Type.NonLeaf);
+                    rowContent += Insert(row[j], "RowItem", childContent);
                 }
-                bodyContent += this.Insert("Row", rowContent, Type.NonLeaf);
+                bodyContent += Insert(row[0], "Row", rowContent);
```

Comment:
got it.

---
In reply to: [77289954](https://github.com/dotnet/docfx/pull/663#discussion_r77289954) [](ancestors = 77289954,77289159)